tests/general/util/pio_tutil.F90

```diff
@@ -149,6 +149,9 @@ SUBROUTINE  PIO_TF_Init_(rearr)
     IF (pio_tf_num_io_tasks_ <= 1 .AND. pio_tf_stride_ > 1) THEN
        pio_tf_stride_ = 1
     END IF
+    IF (pio_tf_stride_ * (pio_tf_num_io_tasks_ - 1) >= pio_tf_world_sz_) THEN
```

Comment:
Maybe we need a comment here stating that "assume root = 0"

Reply:
Right now the testing framework does not allow changing the PIO root, we can refactor this in a future PR that adds that option.